src/test/java/com/assignmentTest/Assignment9Test.java

```diff
@@ -0,0 +1,37 @@
+package com.assignmentTest;
+
+import com.assignment.Assignment9;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class Assignment9Test {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    public void testEvenNumbers() {
+        int min = 1;
+        int max = 500;
+        Assignment9 assignment9 = new Assignment9();
+
+        ArrayList evenNumbers = assignment9.getEvenNumber(min, max);
+        Iterator iterator = evenNumbers.iterator();
+        while (iterator.hasNext()) {
+            logger.info((iterator.next()).toString());
+        }
+
+        /*
+         * Assertion to check if even numbers are retrieved
+         */
```

Comment:
/** = put this before method and then press enter, below is the perfect format

/**
 * 
 */